pkg/apis/operator/v1alpha1/validation/validation.go

```diff
@@ -154,15 +178,34 @@ func validateDNS(dns *operatorv1alpha1.DNSManagement, fldPath *field.Path) field
 func validateRuntimeCluster(dns *operatorv1alpha1.DNSManagement, runtimeCluster operatorv1alpha1.RuntimeCluster, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 
-	if cidrvalidation.NetworksIntersect(runtimeCluster.Networking.Pods, runtimeCluster.Networking.Services) {
-		allErrs = append(allErrs, field.Invalid(fldPath.Child("networking", "services"), runtimeCluster.Networking.Services, "pod network of runtime cluster intersects with service network of runtime cluster"))
+	for i, nodes := range runtimeCluster.Networking.Nodes {
```

Comment:
You could put this loop to the end of the validation function. This ensures that pods and services cidrs are already validated when testing for `NetworksIntersect`.
The sequence could be...
```
	for i, services := range runtimeCluster.Networking.Services {
        ...
        }

	for i, pods := range runtimeCluster.Networking.Pods {
        ...
        }

        for i, nodes := range runtimeCluster.Networking.Nodes {
        ...
        }
```

Reply:
As discussed offline, addressed with https://github.com/gardener/gardener/pull/11251/commits/b6deee9ebc11d57d3fa77ff3adce91a6f0bd682a.
PTAL.